libr/include/r_anal.h

```diff
@@ -576,15 +576,13 @@ typedef struct r_anal_switch_obj_t {
 	RList *cases;
 } RAnalSwitchOp;
 
-#define RANAL void*
-//struct r_anal_t*
-#define RANAL_BLOCK void*
-//struct r_anal_bb_t*
+struct r_anal_t;
```

Comment:
i think this was causing problems in some compilers, but it's green, so i guess it's fine now